cluster/addons/cluster-loadbalancing/glbc/README.md

```diff
@@ -0,0 +1,57 @@
+
+This cluster addon is composed of:
+* A [Google L7 LoadBalancer Controller](https://github.com/kubernetes/contrib/tree/master/Ingress/controllers/gce)
+* A [404 default backend](https://github.com/kubernetes/contrib/tree/master/404-server) Service + RC
+
+It relies on the [Ingress resource](../../../../docs/user-guide/ingress.md) only available in Kubernetes version 1.1 and beyond.
+
+## Quota
+
+GLBC is not aware of your GCE quota. As of this writing users get 3 [GCE Backend Services](https://cloud.google.com/compute/docs/load-balancing/http/backend-service) by default. If you plan on creating Ingresses for multiple Kubernetes Services, remember that each one requires a backend service, and request quota. Should you fail to do so the controller will poll periodically and grab the first free backend service slot it finds. You can view your quota:
+
+```console
+$ gcloud compute project-info describe --project myproject
+```
+See [GCE documentation](https://cloud.google.com/compute/docs/resource-quotas#checking_your_quota) for how to request more.
+
+## Disabling GLBC
+
+Since GLBC runs as a cluster addon, you cannot simply delete the RC. The easiest way to disable it is to do as follows:
+
+* If you want to delete all external loadbalancer resources, first hit the /quit endpoint on the loadbalancer pod:
+
+```console
+$ kubectl get pods --namespace=kube-system
+NAME                                               READY     STATUS    RESTARTS   AGE
+glbc-7bb21                                         1/1       Running   0          1h
+$ kubectl exec glbc-7bb21 curl http://localhost:8081/quit --namespace=kube-system
+$ kubectl logs glbc-7b221 --follow
+...
+I1007 00:30:00.322528       1 main.go:160] Handled quit, awaiting pod deletion.
+```
+Now the pod has been deactivated, but if something kills it, the RC will just recreate it.
+
+* Deactivate the RC
+```console
+$ kubectl edit rc glbc --namespace=kube-system
+```
+
+And add a nodeSelector that is *not* on any node in your cluster:
+```yaml
+...
+spec:
+  nodeSelector:
+    role: loadbalancer
+```
+
+This will cause the RC to wait till you've labelled a node with `role=loadbalancer` before starting a new pod. Note if you want to retain your external loadbalancers but de-activate the controller, simply modifying the RC and deleting the pod will do.
+
+## Limitations
+
+* This cluster addon is still in the Beta phase. It behooves you to read through the GLBC documentation mentioned above and make sure there are no surprises.
+* The recommended way to tear down a cluster with active Ingresses is to either delete each Ingress, or hit the /quit endpoint on GLBC as described below, before invoking a cluster teardown script (eg: kube-down.sh). You will have to manually cleanup GCE resources through the [cloud console](https://cloud.google.com/compute/docs/console#access) or [gcloud CLI](https://cloud.google.com/compute/docs/gcloud-compute/) if you simply tear down the cluster with active Ingresses.
```

Comment:
I would make /quit more obviously destructive.  Something like /delete-everything or /nukular - make it clear that it's not quitting the app, it's quitting the whole service.